analytics/tools/patch_logfile.ipynb

```diff
@@ -0,0 +1,191 @@
+{
+ "cells": [
```

Comment:
Can we add a brief comment on what this logfile patches (e.g, moves most recent model one forward)?